tests/ui/ReportListItemHeaderTest.tsx

```diff
@@ -25,7 +25,6 @@ const mockSearchContext = {
     selectedReports: {},
     setSelectedReports: jest.fn(),
     selectedTransactionIDs: [],
-    setSelectedTransactionIDs: jest.fn(),
     clearSelectedItems: jest.fn(),
```

Comment:
Also please mock all missing values

```suggestion
    clearSelectedTransactions: jest.fn(),
```